sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -323,6 +386,108 @@ def __repr__(self):
         )
 
 
+class HealthcareEntity(DictMixin):
+    """HealthcareEntity contains information about a Healthcare entity found in text.
+
+        :ivar str text: Entity text as appears in the request.
+        :ivar str category: Entity category, such as Dosage or MedicationName, etc.
+        :ivar str subcategory: Entity subcategory.  # TODO: add subcategory examples
+        :ivar int offset: The Healthcare entity text offset from the start of the document.
+        :ivar int length: The length of the Healthcare entity text.
+        :ivar float confidence_score: Confidence score between 0 and 1 of the extracted
+            entity.
+        :ivar links: A collection of entity references in known data sources.
+        vartype links: list[~azure.ai.textanalytics.HealthcareEntityLink]
+    """
+
+    def __init__(self, **kwargs):
+        self.text = kwargs.get("text", None)
+        self.category = kwargs.get("category", None)
+        self.subcategory = kwargs.get("subcategory", None)
+        self.offset = kwargs.get("offset", None)
+        self.length = kwargs.get("length", None)
```

Comment:
this is just something to keep in mind, but for 5.1, we removed the `length` properties since they're always returned with `text`, and if users want the length of text, they can just do `len(text)`. Not sure how this fits in with preview 1 (i.e., is this a "design change" we'll include), but wanted to give you that context

Reply:
Ok, I've removed this from the HealthcareEntity class.  I also updated the samples to remove references.  There were no uses of this in the tests, so no update was needed there.